src/Glpi/Form/Destination/FormDestination.php

```diff
@@ -293,38 +286,6 @@ public function getConcreteDestinationItem(): ?FormDestinationInterface
         return new $class();
     }
 
-    /**
-     * Get valid destinations for a given form
-     *
-     * @param Form $form
-     *
-     * @return AbstractFormDestinationType[]
-     */
-    protected function getDestinationsForForm(Form $form): array
```

Comment:
This method was unused and its code was wrong (probably a remnant from an old re-factorization).